java/offline-geocode/README.md

```diff
@@ -1,17 +1,26 @@
-# Offline Geocode
-Geocode addresses to locations and reverse geocode locations to addresses when the device does not have network connectivity.
+# Offline geocode
 
-![Offline Geocode App](offline-geocode.png)
+Geocode addresses to locations and reverse geocode locations to addresses offline.
+
+![Image of offline geocode](offline-geocode.png)
+
+## Use case
+
+You can use an address locator file to geocode addresses and locations. For example, you could provide offline geocoding capabilities to field workers repairing critical infrastructure in a disaster when network availability is limited.
 
 ## How to use the sample
-Type the address in the Search menu option or select from the list to `Geocode` the address and view the result on the map. Long-press on the location you want to Reverse Geocode. Selecting the output pin enables real-time reverse geocoding. Select the pin to highlight the `PictureMarkerSymbol` (i.e. single tap on the pin) and then tap-hold and drag on the map to get real-time geocoding.
+
+Type the address in the Search menu option or select from the list to `Geocode` the address and view the result on the map. Tap the location you want to reverse geocode. Select the pin to highlight the `PictureMarkerSymbol` (i.e. single tap on the pin) and then tap-hold and drag on the map to get real-time geocoding.
```

Comment:
I don't know if it's just my end, but I can _only_ select from a predetermined list of addresses. Typing or tapping have no effect